ai-platform/snippets/test/code-model-tuning.test.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable */
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it} = require('mocha');
+const uuid = require('uuid')
+const sinon = require('sinon');
+
+const aiplatform = require('@google-cloud/aiplatform');
+const clientOptions = {
+  apiEndpoint: 'europe-west4-aiplatform.googleapis.com',
```

Comment:
**nit:** Swap hardcoded reference to `europe-west4` and add in `${location}`. Just so only one ref needs to change if this sample needs to be updated to another location.

Reply:
Changed. Thanks.